CMakeLists.txt

```diff
@@ -331,7 +331,7 @@ endif ()
 
 message(STATUS "Build Type: ${CMAKE_BUILD_TYPE}")
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wno-strict-aliasing")
 
```

Comment:
Looks like we should be compiling with `-fno-strict-aliasing`; I think this is considered a best practice in general to avoid odd edge cases, even though it may prevent certain compiler optimizations